protocol/app/msgs/app_injected_msgs_test.go

```diff
@@ -51,7 +51,9 @@ func TestAppInjectedMsgSamples_GetSigners(t *testing.T) {
 		_ = testTxBuilder.SetMsgs(sample.Msg)
 		sigTx, ok := testTxBuilder.GetTx().(authsigning.SigVerifiableTx)
 		require.True(t, ok)
-		require.Empty(t, sigTx.GetSigners())
+		signers, err := sigTx.GetSigners()
+		require.Error(t, err)
```

Comment:
Fixed.

That was supposed to be NoError and it was causing an error since the MakeTestEncodingConfig wasn't using the correct InterfaceRegistry since it was using the default Cosmos one that doesn't have the custom signers defined.